tools/subset.py

```diff
@@ -145,7 +146,7 @@ def subset_font_raw(font_in, font_out, unicodes, opts):
     if pe:
         print('Generate("' + font_out + '")', file=pe)
         pe.close()
-        os.system("fontforge -script " + pe_fn)
+        subprocess.call("fontforge -script " + pe_fn, shell=True)
```

Comment:
Make this a list and remove `shell=True` like the others (`pe_fn` is just a single file name).